src/utils/Streami18n.js

```diff
@@ -243,6 +243,7 @@ export class Streami18n {
       fallbackLng: false,
       debug: finalOptions.debug,
       lng: this.currentLanguage,
+      interpolation: { escapeValue: false },
```

Comment:
This is to avoid following issue:

e.g., of there is inverted comma in user name, then typing string becomes - `Jake O&#39;Tester is typing`.